src/Altinn.Apps/KubernetesWrapper/Controllers/DeploymentsController.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using KubernetesWrapper.Services.Interfaces;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace KubernetesWrapper.Controllers
+{
+    /// <summary>
+    ///  Controller containing all actions related to kubernetes deployments
+    /// </summary>
+    [Route("/[controller]")]
+    [ApiController]
+    public class DeploymentsController : ControllerBase
+    {
+        private readonly IKubernetesAPIWrapper _apiWrapper;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DeploymentsController"/> class
+        /// </summary>
+        /// <param name="apiWrapper">The kubernetes api wrapper client</param>
+        public DeploymentsController(IKubernetesAPIWrapper apiWrapper)
+        {
+            _apiWrapper = apiWrapper;
+        }
+
+        /// <summary>
+        /// Get a list of deployments. For a more detailed spec of parameters see Kubernetes API DOC
+        /// </summary>
+        /// <param name="labelSelector">A selector to restrict the list of returned objects by their labels. Defaults to everything.</param>
+        /// <param name="fieldSelector">A selector to restrict the list of returned objects by their fields. Defaults to everything.</param>
+        /// <returns>A list of deployments in the cluster</returns>
+        [HttpGet]
+        [EnableCors]
```

Comment:
Is it meant to be accessed from outside of altinn studio?

Reply:
Yes, this is just to enable our react-app to be able to call the app. In the future the altinn-platform would probably work as an intermediary, but probably not in mvp2.